eZ/Publish/API/Repository/Values/ContentType/FieldDefinition.php

```diff
@@ -113,4 +113,11 @@
      * @var bool
      */
     protected $isSearchable;
+
+    /**
+     * Based on mainLanguageCode of contentType.
+     *
+     * @var string
+     */
+    protected $mainLanguageCode;
```

Comment:
I missed one thing, for better DX we also update PhpDoc, when adding read-only property, e.g.:
```php
/**
 * (...)
 * @property-read $mainLanguageCode main Translation (language code) of a multilingual Field Definition
 */
```